index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="card">
+            <div class="circle">
+                <h2>01</h2>
+            </div>
+            <div class="content">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nihil voluptatibus recusandae perspiciatis cupiditate qui quisquam reiciendis explicabo laboriosam deserunt nesciunt voluptatum dicta quasi, amet eligendi ipsam molestias blanditiis repudiandae enim.</p>
```

Comment:
you can use real data too, such as 3 cards each for html, css & js, describing their use or definition